local_db/migrations/0015_update_manifest_data.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 5.0.4 on 2024-05-07 12:46
+
+from django.db import migrations
+
+from airlock.business_logic import BusinessLogicLayer, Workspace
```

Comment:
This might be misplaced anxiety on my part but I always get a bit nervous about db migrations which pull in a lot of application logic because they stick around for a long time during which the application logic tends to evolve.

Another option is to have a one-off, "data backfill" commands that you can generally delete after they've been used.

Reply:
I dithered about which way to do it, and thought the migration would be simpler, and would get squashed soon anyway in #249.
Although, I now notice that some things changed since I wrote the data migration, so it won't work properly now anyway, so I might as well move it to a separate script

Reply:
> thought the migration would be simpler, and would get squashed soon anyway in #249.
Oh yeah, that is a fair point. Fine to leave it where it is then if that's easier.

Reply:
Ah, and I just noticed a bug in the test factories that generate `manifest.json`s which has meant that I've made it lookup absolute paths when they should be relative.

Reply:
I've made it a temporary backpopulate command so that I can also add a test for it (which would have found the   factories bug if I'd written it in the first place). Also it'll be easier to rerun when we need to add in the repo, when that's available on the manifest.json